axiom/tests/test_cmd_appinfo.c

```diff
@@ -941,33 +942,39 @@ static void test_process_event_harvest_config(void) {
       "}"
       "}");
 
+  nr_app_info_t info;
+  info.span_events_max_samples_stored
```

Comment:
Does the rest of the var 'info' need to be initialized?  I believe a local struct will not be initialized to any predictable values in general.

Reply:
Generally yes, but in this test case ONLY `info.span_events_max_samples_stored` is being used, so the remaining values aren't needed.